.github/workflows/docker.yml

```diff
@@ -1,7 +1,6 @@
 name: Create and publish docker images
 
 on:
-  pull_request:
```

Comment:
Shouldn't we keep this for validation purpose? (As I understand it it is slightly different from push as it will run the workflow on a rebased version of the branch)

Reply:
For me, every time we push to a remote branch (with or without a Pull request), it creates a docker image. With this specification, we build twice the docker image when we push on a branch with an open PR. It looks like a quick win but we can think about how to change the way we build docker image

Reply:
I think the two workflows are different:
 - *pull* will run the workflow on the current state of the branch
 - *pull_request* will simulate the state of master branch after merging
 
 From https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request:
 > Note that GITHUB_SHA for this event is the last merge commit of the pull request merge branch. If you want to get the commit ID for the last commit to the head branch of the pull request, use github.event.pull_request.head.sha instead.

Reply:
So I'd say that it is worth keeping this activated as it could prevent some edge cases where merging a branch would break a workflow.